pkg/frameworkPackage.targets

```diff
@@ -43,6 +43,15 @@
       <Version>$(PlatformPackageVersion)</Version>
       <TargetFramework>$(TargetFramework)</TargetFramework>
     </Dependency>
+
+    <!-- Include the update to v1 lineup package, which now has no runtime package mappings.
+         This will reduce the number of packages downloaded when mixing v1 and later
+         packages as well as reduce our dependence on conflict resolution for dropping
+         those v1 runtime packages -->
+    <Dependency Condition="'$(IsLineupPackage)' == 'true'" Include="Microsoft.NETCore.Targets">
+      <Version>2.0.0</Version>
```

Comment:
Any reason to not be a ProjectReference as opposed to hardcoding the version? Also is there much reason to reference this in the private transport packages?